tests/ui/resolve/issue-18252.stderr

```diff
@@ -3,6 +3,11 @@ error[E0533]: expected value, found struct variant `Foo::Variant`
    |
 LL |     let f = Foo::Variant(42);
    |             ^^^^^^^^^^^^ not a value
+   |
+help: you might have meant to create a new value of the struct
+   |
+LL |     let f = Foo::Variant { x: /* value */ };
```

Comment:
Suggesting `Foo::variant { x: 42 }` would be nice here, if it's easy.

Reply:
I don't think it's *easy* per-se. It's doable, but I'd prefer doing it as part of its own PR to avoid landing a larger change than strictly necessary.